collection/update_specifications.go

```diff
@@ -50,13 +50,15 @@ func (dc *Collection) UpdateSpecifications(index string, collection string, spec
 	query := &types.KuzzleRequest{
 		Controller: "collection",
 		Action:     "updateSpecifications",
-		Body:       jsonBody,
+		Body:       json.RawMessage(jsonBody),
 	}
 
 	go dc.Kuzzle.Query(query, options, ch)
 
 	res := <-ch
 
+	fmt.Printf("%s", res.Error.Error())
```

Comment:
`A wild forgotten Printf appear`

Reply:
https://youtu.be/XE6YaLtctcI

Reply:
![tenor](https://user-images.githubusercontent.com/7868838/46721787-3018bb80-cc74-11e8-8d1d-b084c0cd6613.gif)